### ISSUE TITLE
Stream Deck mk.2 not working on Ubuntu 21.04

### ISSUE THREAD

**Issue by maschmann** [timothycrosley/streamdeck-ui#147]
Hi,

first of all: Great work with the UI :-) Thank you for the great effort of providing this for free.

I've got a brand new mk.2 here, previously used the first one (15 keys) and it worked fine with streamdeck-ui (1.1.2 from pip). The new one is not detected.
OS is Ubuntu 21.04, default kernel on a Thinkpad T495s. 
So I checked for the ProdID, which is not in the udev list -> 0080
Added and tried again to no avail.

```
T:  Bus=02 Lev=02 Prnt=03 Port=03 Cnt=02 Dev#=  5 Spd=480 MxCh= 0
D:  Ver= 2.00 Cls=00(>ifc ) Sub=00 Prot=00 MxPS=64 #Cfgs=  1
P:  Vendor=0fd9 ProdID=0080 Rev=02.00
S:  Manufacturer=Elgato
S:  Product=Stream Deck MK.2
S:  SerialNumber=XXXXXXXXXX
C:  #Ifs= 1 Cfg#= 1 Atr=a0 MxPwr=500mA
I:  If#=0x0 Alt= 0 #EPs= 2 Cls=03(HID  ) Sub=00 Prot=00 Driver=usbhid
```
Syslog is not as helpful as I though when having streamdeck-ui running and plugging the device in:

```
Jul 26 13:32:48 deepthought kernel: [  841.522421] usb 2-4.4: new high-speed USB device number 10 using xhci_hcd
Jul 26 13:32:48 deepthought kernel: [  841.672843] usb 2-4.4: New USB device found, idVendor=0fd9, idProduct=0080, bcdDevice= 2.00
Jul 26 13:32:48 deepthought kernel: [  841.672849] usb 2-4.4: New USB device strings: Mfr=1, Product=2, SerialNumber=3
Jul 26 13:32:48 deepthought kernel: [  841.672852] usb 2-4.4: Product: Stream Deck MK.2
Jul 26 13:32:48 deepthought kernel: [  841.672853] usb 2-4.4: Manufacturer: Elgato
Jul 26 13:32:48 deepthought kernel: [  841.672855] usb 2-4.4: SerialNumber: XXXXXXXXX
Jul 26 13:32:48 deepthought kernel: [  841.732090] retire_capture_urb: 15 callbacks suppressed
Jul 26 13:32:48 deepthought kernel: [  841.764604] input: Elgato Stream Deck MK.2 as /devices/pci0000:00/0000:00:08.1/0000:05:00.3/usb2/2-4/2-4.4/2-4.4:1.0/0003:0FD9:0080.0010/input/input51
Jul 26 13:32:48 deepthought kernel: [  841.822601] hid-generic 0003:0FD9:0080.0010: input,hidraw1: USB HID v1.10 Device [Elgato Stream Deck MK.2] on usb-0000:05:00.3-4.4/input0
Jul 26 13:32:48 deepthought mtp-probe: checking bus 2, device 10: "/sys/devices/pci0000:00/0000:00:08.1/0000:05:00.3/usb2/2-4/2-4.4"
Jul 26 13:32:48 deepthought mtp-probe: bus: 2, device: 10 was not an MTP device
Jul 26 13:32:48 deepthought upowerd[1101]: treating change event as add on /sys/devices/pci0000:00/0000:00:08.1/0000:05:00.3/usb2/2-4/2-4.4
Jul 26 13:32:48 deepthought mtp-probe: checking bus 2, device 10: "/sys/devices/pci0000:00/0000:00:08.1/0000:05:00.3/usb2/2-4/2-4.4"
Jul 26 13:32:48 deepthought mtp-probe: bus: 2, device: 10 was not an MTP device
```

If I can provide you with more information, let me know.

**Comment by maschmann** [comment]
Ok, this has several components - seems the underlying api is already adjusted, so you only need to add the new device/productId to the documentation as soon as the new version of streamdeck is released:  https://github.com/abcminiuser/python-elgato-streamdeck/issues/73

**Comment by dodgyrabbit** [comment]
I've merged #154 which adds the correct rules and also committed the dependency on streamdeck v0.8.5.
Closing this issue as this should all work on the next release.
